README.md

```diff
@@ -26,20 +26,17 @@ To preview the Lua API docs locally visit [MkDocs](https://www.mkdocs.org/) and
 #### Windows
 	./bootstrap-vcpkg.bat
 	[OPTIONAL] ./vcpkg integrate install
-	./vcpkg install asio bullet3 glew glfw3 glm lua openal-soft protobuf zlib spdlog imgui sol2
```

Comment:
You can't just not do this step, manual setup doesn't work without it (or something else)

Reply:
To expand on this a little, I think you should stipulate a minimum required version of vcpkg or an explanation of how the full setup process should work.